export.py

```diff
@@ -95,19 +142,14 @@ def json_to_markdown(json):
 
 def group_comments_by_post(comments):
     posts = {}
-
```

Comment:
I value my newlines, they highlight the structure of the code. Removing them turns the structure into a wall of text, please undo this.

Reply:
Done and done!

Reply:
Went through and tried to recover other newlines and your original comments.